### DISCUSSION TITLE
post/2024-07-10-angular-todo-mvc-with-spring-boot-podman-openshift/

### DISCUSSION THREAD

**Discussion by giscus** [sshaaf/sshaaf.github.io#2] [Announcements]
# post/2024-07-10-angular-todo-mvc-with-spring-boot-podman-openshift/

This article guides you through building a Spring Boot demo application and deploying it on Kubernetes using the JBoss Web Server Operator. The application uses the TodoMVC Angular front-end, integrated with a Spring Boot backend. The Todo entity is defined with JPA annotations for database mapping. The TodoController handles CRUD operations, and TodoRepository extends JpaRepository for database interactions. The application can be run locally or packaged with Tomcat as an embedded server.

https://shaaf.dev/post/2024-07-10-angular-todo-mvc-with-spring-boot-podman-openshift/?ref=dailydev

**Comment by Fatim94** [comment]
Thank you for sharing.
